### PR TITLE
Added follower/runner & libraries

### PR DESCRIPTION
**Pull request by lewislepton**
new episodes

### CODE REVIEW

**Comment by Simn** [comment]
Something doesn't parse

**Comment by lewislepton** [comment]
hmmm. weird. ill have another go once I get this new episode edited. I did clone again & started fresh. so ill try again & see in a bit ;)

File: videos/tutorials/4.kha-tutorial-series.json [hunk 1]
```diff
@@ -615,6 +612,30 @@
...
...description": "SUPPORT ON PATREON FROM AS LITTLE AS $1:<br/><a href='https://patreon.com/lewislepton'>https://patreon.com/lewislepton</a><br/><br/>we can move around, we also create other entities, but what about something following us?<br>we look at follower AI<br/><br/>source code:<br/><a href='https://github.com/lewislepton/kha-tutorial-series'>https://github.com/lewislepton/kha-tutorial-series</a><br/><br/>intro & outro music is from this EP:<br/><a href='https://lewislepton.bandcamp.com/album/coat-noise-ep'>https://lewislepton.bandcamp.com/album/coat-noise-ep</a><br/><br/>like, share & subscribe to stay up to date.<br/><br/><a href='https://lewislepton.com'>https://lewislepton.com</a><br/><a href='https://patreon.com/lewislepton'>https://patreon.com/lewislepton</a><br/><a href='https://twitter.com/lewislepton'>https://twitter.com/lewislepton</a><br/><a href='https://facebook.com/lewislepton'>https://facebook.com/lewislepton</a><br/><a href='https://lewislepton.bandcamp.com'>https://lewislepton.bandcamp.com</a><br/><a href='https://soundcloud.com/lewislepton'>https://soundcloud.com/lewislepton</a><br/><a href='https://github.com/lewislepton'>https://github.com/lewislepton</a>",
```
Comment (ibilon) [inline videos/tutorials/4.kha-tutorial-series.json]: The error is from the unclosed br at `about something following us?<br>we look at follower AI`,
sorry about the unfriendly parser.

**Comment by lewislepton** [comment]
ok. changed slightly so hopefully that will do the trick

File: videos/tutorials/4.kha-tutorial-series.json (hunk 1 above)
Comment (lewislepton) [reply videos/tutorials/4.kha-tutorial-series.json]: nay worries. ive just made the changes. so hopefully that fixes it. cheers ;)

**Pull request closed without merging**